analysis/study_utils.py

```diff
@@ -181,14 +182,14 @@ def calculate_rate(df, value_col, rate_per=1000, round_rate=False):
 
     return rate
 
+
 def round_values(x, base=5):
-    if not np.isnan(x):
-        rounded = int(base * round(x/base))
+    if x is not None and not str(x).startswith("<=") and not np.isnan(x):
```

Comment:
@LFISHER7 This seems to work with the tests, although it's really messy. Can you confirm if that's correct? If so we should be able to refactor it, maybe to check if x is numerical?

Reply:
That looks correct. Refactoring to check whether x is numerical makes sense to me